dataprep-pipelines/tests/integration.rs

```diff
@@ -3,9 +3,17 @@ use tokio::fs;
 
 use dataprep_pipelines;
 
+
+#[test]
+fn test_macro() {
+    // macro to determine if two directories/files are equal
+    assert_paths!(input_dir, unpacked_dir);
```

Comment:
thanks for the example- can you rebase and edit my integration test to use this?